pkgs/applications/printing/pappl/default.nix

```diff
@@ -13,23 +12,18 @@
 
 stdenv.mkDerivation rec {
   pname = "pappl";
-  version = "1.1.0";
+
+  # Also update the meta.changelog entry if you change the version
+  # (it changes for every release)
```

Comment:
```suggestion
```
The purpose of meta.changelog is to display the correct changelog when auto updating. So it doesn't make much sense to update it manually for every release.

Reply:
resolved in rebased commits